packages/runtime/container-runtime/src/pendingStateManager.ts

```diff
@@ -795,7 +794,9 @@ export class PendingStateManager implements IDisposable {
 			assert(batchMetadataFlag !== false, 0x41b /* We cannot process batches in chunks */);
 
 			// The next message starts a batch (possibly single-message), and we'll need its batchId.
-			const batchId = getEffectiveBatchId(pendingMessage);
+			const batchId = pendingMessage.batchInfo.ignoreBatchId
```

Comment:
@dannimad - This is the other fix.  If we end up resubmitting an ID Allocation batch, it shouldn't get batchId.  Otherwise it results in a useless empty batch